README.md

```diff
@@ -35,6 +35,12 @@ sudo apt install \
     cmake extra-cmake-modules gettext libfmt-dev
```

Comment:
Nit: should we just add `libicu-dev` here? Since we are already telling people they need to install those `-dev` packages here…

Reply:
Updated at 84b0dc5053ac9795ec105b91933e126606105e43